crates/shared/src/http_solver/model.rs

```diff
@@ -182,6 +182,20 @@ impl Interaction for InteractionData {
     }
 }
 
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub enum Score {
+    /// The score used for ranking.
+    #[serde(with = "u256_decimal")]
+    Score(U256),
+    /// This option is used to indicate that the solver did not provide a score.
+    /// Instead, the score should be computed by the protocol.
+    /// To have more flexibility, the protocol score can be tweaked by the
+    /// solver by providing a multiplication factor. Expected value: [0,
+    /// inf], 0 being 100% discount, 1 being the same as protocol score
+    MulFactor(f64),
```

Comment:
nit: I would have called this `scoreFactor` - `mulFactor` doesn't have much information.

Reply:
Fixed here https://github.com/cowprotocol/services/pull/1282